pkg/utils/utils_test.go

```diff
@@ -77,3 +77,39 @@ func TestRemoveHypelink(t *testing.T) {
 		}
 	}
 }
+
+func TestStructDump(t *testing.T) {
+	type Thread struct {
+		TimeStamp int64
+		Team      string
+	}
+	type Message struct {
+		Text    string
+		UserID  int
+		Threads []Thread
+	}
+
+	got := StructDumper().Sdump(Message{
+		Text:   "Hello, Botkube!",
+		UserID: 3,
+		Threads: []Thread{
+			{
+				TimeStamp: int64(2344442424),
+				Team:      "MetalHead",
+			},
+		},
+	})
+	expected := `utils.Message{
+  Text: "Hello, Botkube!",
+  UserID: 3,
+  Threads: []utils.Thread{
+    utils.Thread{
+      TimeStamp: 2344442424,
+      Team: "MetalHead",
+    },
+  },
+}`
+	if got != expected {
+		t.Errorf("expected: %s, got: %s", expected, got)
+	}
```

Comment:
we use testify in our tests, so just for the sake of consistency, please do:
```suggestion
	assert.Equal(t, expected, got)
```

Additionally, in case of error we get more valuable message with the diff:
```
--- Expected
+++ Actual
@@ -1,3 +1,3 @@
utils.Message{
-  Text: "Hello, Botkube!s",
+  Text: "Hello, Botkube!",
UserID: 3,
```

---

NOTE: I know that other tests have this `if` statements, but those are the old one, the new one are written with the testify lib, if you don't mind you can also adjust other test in this lib to follow this pattern.